ROADMAP.md

```diff
@@ -8,124 +7,164 @@ Notes: This document is still in review will may be heavily modified based on st
+
+### [A. 📺 Universal Browser Connectivity](https://github.com/libp2p/go-libp2p/issues/1811)
+
+**Why**: A huge part of “the Web” is happening inside the browser. As a universal p2p networking stack, libp2p needs to be able to offer solutions for browser users.
 
-#### Up Next
--   D.2 📺 Universal Browser Connectivity - **[WebTransport](https://github.com/libp2p/go-libp2p/issues/1717)** update to new draft versions
-    - Dependency on Chrome needs to support new draft version of WebTransport protocol (still under development by IETF and W3C)
--  E.2 🧪 Future-proof testing - **Comprehensive testing for non-transport features**
--  G 📢 Judicious Address Advertisements
+**Goal**: go-libp2p ships with up-to-date WebTransport and (libp2p-) WebRTC implementations, enabled by default. This allows connections between browsers and public nodes, browsers and non-public nodes, as well as two browsers.
 
-# Roadmap Appendix
-## [A.⚡️Handshakes at the Speed of Light](https://github.com/libp2p/go-libp2p/issues/1807)
+#### 1. [WebRTC: Browser to Server](https://github.com/libp2p/go-libp2p/pull/1655)
+Add support for WebRTC transport in go-libp2p, enabling browser connectivity with servers. This will cover the browsers that don't support WebTransport (most notable is iOS Safari). This is getting close to finalized.
+#### 2. WebRTC: Browser to Browser
+A follow up to A.1 where we will begin the work to specify the semantics of browser to browser connectivity and then implement it in go-libp2p.
+#### 3. [WebTransport: Update to new draft versions](https://github.com/libp2p/go-libp2p/issues/1717)
+As the protocol is still under development by IETF and W3C, the go-libp2p implementation needs to follow. We have a dependency on Chrome to support the new draft version of WebTransport protocol. To stay up to date, we will have to move as soon as Chrome ships supports the new draft version.
+
+### [B. ⚡️ Handshakes at the Speed of Light](https://github.com/libp2p/go-libp2p/issues/1807)
 
 **Why**: Historically, libp2p has been very wasteful when it comes to round trips spent during connection establishment. This is slowing down our users, especially their TTFB (time to first byte) metrics.
 
 **Goal**: go-libp2p optimizes its handshake latency up to the point where only increasing the speed of light would lead to further speedups. In particular, this means:
 
-1. [Early Muxer Negotiation](https://github.com/libp2p/specs/issues/426): cutting off the 1 RTT wasted on muxer negotiation
-2. [Adding security protocol](https://github.com/libp2p/specs/pull/353): cutting off the 1 RTT wasted on security protocol negotiation by including the security protocol in the multiaddr
-3. 0.5 RTT data optimization: using 0.5-RTT data (for TLS) / a Noise Extension to ship the list of Identify protocols, cutting of 1 RTT that many protocols spend waiting on `IdentifyWait`
+#### 1. [Early Muxer Negotiation](https://github.com/libp2p/specs/issues/426)
```

Comment:
Slight preference for the list instead of the headers. But not strong enough to ask for a change.